packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -282,6 +282,7 @@ export const messages = {
   org_select_text: 'Select an org to set as default',
   org_expired: 'Expired',
   missing_default_org: 'No Default Org Set',
+  default_org_expired: 'Your default org is expired.',
```

Comment:
@sbudhirajadoc and for this one. This is for the warning notification

Reply:
Your default org has expired. Select a new Salesforce org as your default org try again.

Reply:
<img width="598" alt="image" src="https://github.com/user-attachments/assets/e642a327-99a5-496b-abb2-96ed50978f30">

Reply:
final -- Your default org has expired. Some of the the command palette commands may not longer work. Try switching your default org and try again.